intbot/core/bot/channel_router.py

```diff
@@ -0,0 +1,97 @@
+"""
+Channel router that decides where to send particular message
+"""
+
+from dataclasses import dataclass
+
+from core.integrations.github import (
+    GithubProjects,
+    GithubRepositories,
+    parse_github_webhook,
+)
+from core.models import Webhook
+from django.conf import settings
+
+
+@dataclass
+class DiscordChannel:
+    channel_id: str
+    channel_name: str
+
+
+dont_send_it = DiscordChannel(channel_id="0", channel_name="/dev/null")
+
+
+class Channels:
+    test_channel = DiscordChannel(
+        channel_id=settings.DISCORD_TEST_CHANNEL_ID,
+        channel_name=settings.DISCORD_TEST_CHANNEL_NAME,
+    )
+
+    board_channel = DiscordChannel(
+        channel_id=settings.DISCORD_BOARD_CHANNEL_ID,
+        channel_name=settings.DISCORD_BOARD_CHANNEL_NAME,
+    )
+    ep2025_channel = DiscordChannel(
+        channel_id=settings.DISCORD_EP2025_CHANNEL_ID,
+        channel_name=settings.DISCORD_EP2025_CHANNEL_NAME,
+    )
+    em_channel = DiscordChannel(
+        channel_id=settings.DISCORD_EM_CHANNEL_ID,
+        channel_name=settings.DISCORD_EM_CHANNEL_NAME,
+    )
+    website_channel = DiscordChannel(
+        channel_id=settings.DISCORD_WEBSITE_CHANNEL_ID,
+        channel_name=settings.DISCORD_WEBSITE_CHANNEL_NAME,
+    )
+    bot_channel = DiscordChannel(
+        channel_id=settings.DISCORD_BOT_CHANNEL_ID,
+        channel_name=settings.DISCORD_BOT_CHANNEL_NAME,
+    )
+
+
+def discord_channel_router(wh: Webhook) -> DiscordChannel:
+    if wh.source == "github":
+        return github_router(wh)
+
+    elif wh.source == "internal":
+        return internal_router(wh)
+
+    return dont_send_it
```

Comment:
```suggestion
def discord_channel_router(wh: Webhook) -> DiscordChannel:
    match wh.source:
        case "github":
            return github_router(wh)
        case "internal":
            return internal_router(wh)
        case _:
            return null_channel 
```

Reply:
I can replace this with the dictionary lookup like in the example below - it would work better than match.